contribs/gmf/src/directives/objectediting.js

```diff
@@ -685,6 +714,7 @@ gmf.ObjecteditingController.prototype.resetGeometryChanges_ = function() {
   if (this.geometryChanges_.length === 0) {
     const geometry = this.feature.getGeometry();
     const clone = gmf.ObjecteditingController.cloneGeometry_(geometry);
+    goog.asserts.assert(clone);
```

Comment:
More specifically, this line broke the tool.  A `null` geometry is valid when we have a feature without any geom, i.e. a new feature.